oasis-webapp/src/main/resources/templates/Login.soy

```diff
@@ -120,11 +124,10 @@ aside a { display: inline-block; padding: 5px 15px; font-weight: lighter; text-d
         {/if}
         <div class=btn-row>
           <button>
-            <img alt="" src="/images/go.png" width=50 height=50>
             {msg desc="Form submit button"}
               Log in
             {fallbackmsg desc="Form submit button"}
-              Go!
+              Go !
```

Comment:
No, English doesn't put space before punctuation like French.
Moreover, this is translatable content so it must not be changed without following a precise process.

Actually, because it's a `fallbackmsg` here and the `msg` has been translated in all supported languages, we can simply get rid of it (same for "Sign up" below)